app/helpers/heatmap_helper.rb

```diff
@@ -216,7 +150,7 @@ def self.fetch_top_taxons(
       #{rpm_sql} AS rpm,
       #{zscore_sql} AS zscore
     FROM taxon_counts
-    LEFT OUTER JOIN pipeline_runs pr ON pipeline_run_id = pr.id
+    JOIN pipeline_runs pr ON pipeline_run_id = pr.id
```

Comment:
I thought this was needed earlier but now I know that the pipeline runs should exist

Reply:
This is guaranteed by the foreign keys?

Reply:
No this is not guaranteed by the foreign keys because we explicitly decided to exclude `taxon_counts` but I know now that we don't intentionally delete pipeline_runs and leave behind their taxon_counts.

Reply:
In case of human/bug error, I guess the effect here would be to remove all taxon counts that do not have a pipeline run, which seems fine. We should probably report any such findings somehow, no?

Can replace JOIN by INNER JOIN ? I had to look it up because I was not fully sure which one was the default :)

Reply:
Yes, sure thing